packages/core/src/tracing/opentracing/Span.js

```diff
@@ -53,7 +53,7 @@ function Span(tracer, name, fields) {
     data: {
       service: serviceName,
       sdk: {
-        type: 'local',
+        type: 'entry',
```

Comment:
Would it work if we did `type: parentContext ? 'local' : 'entry'`?
It's sort of personal preference, but on the other hand, seems easier to read, as one doesn't need to read the code further to realize that the `type` may change based on whether `parentContext` exists.
But again, this is mostly personal preference.

Reply:
Absolutely, good call.